clients/unshaded/src/test/java/tachyon/client/file/options/CreateOptionsTest.java

```diff
@@ -60,7 +54,7 @@ public void defaultsTest() {
     UnderStorageType ufsType = UnderStorageType.SYNC_PERSIST;
     conf.set(Constants.USER_BLOCK_SIZE_BYTES_DEFAULT, "64MB");
     conf.set(Constants.USER_FILE_UNDER_STORAGE_TYPE_DEFAULT, ufsType.toString());
-    Whitebox.setInternalState(ClientContext.class, "sTachyonConf", conf);
+    ClientContext.reset(conf);
```

Comment:
We'll eventually want to remove the `ClientContext` singleton and pass `ClientContext`s around instead. That way we won't have to mock and global state to change the config for a single test. For now I'll reset the configurations.